src/resources/views/components/color-picker.blade.php

```diff
@@ -80,10 +87,11 @@ class="h-full rounded-r-md"
                          :style="{ 'background-color': color.value }"
                          x-on:click="select(color)"
                          :title="color.name"
+                         :focusable-id="color.value + color.name"
                          type="button"
                      ></button>
                  </template>
              </div>
-        </x-wireui::parts.popover>
+        </x-wireui::parts.popover2>
```

Comment:
`popover2`, don't we put other name?

Reply:
New popover component, until all classes are migrated. Then, I'll delete the previous one and rename this.